docs/docs/user-guide/qa.md

```diff
@@ -0,0 +1,84 @@
+# Quality Assurance
+
+## Overview
+
+Browsertrix Crawler has the capability to "re-crawl" an existing crawl to compare what the browser encountered on a website during crawling against the replay of the crawl WACZ. The WACZ produced by this analysis run includes additional comparison data (stored as WARC `resource` records) for the pages found during crawling against their replay in ReplayWeb.page along several dimensions, including screenshot, extracted text, and page resource comparisons.
+
+!!! note
+
+    QA features described on this page are available in Browsertrix Crawler releases 1.1.0 and later.
+
+
+## Getting started
+
+To be able to run QA on a crawl, we must first have an existing crawl, for example:
+
+```sh
+docker run -v $PWD/crawls:/crawls/ -it webrecorder/browsertrix-crawler crawl --url https://webrecorder.net/ --collection example-crawl --text to-warc --screenshot view --generateWACZ```
+
+Note that this crawl must be run with `--generateWACZ` flag as QA requires a WACZ to work with, and also ideally the `--text to-warc` and `--screenshot view` flags as well (see below for more details on comparison dimensions).
+
+To analyze this crawl, call Browsertrix Crawler can be run with the `qa` entrypoint, passing the original crawl WACZ as the `qaSource`:
+
+```sh
+docker run -v $PWD/crawls/:/crawls/ -it webrecorder/browsertrix-crawler qa --qaSource /crawls/collections/example-crawl/example-crawl.wacz --collection example-qa --generateWACZ
+```
+
+The `qaSource` can be:
+- A local WACZ file path or a URL
+- A single WACZ or a JSON file containing a list of WACZ files in the `resources` json (Multi-WACZ)
+
+This assumes an existing crawl that was created in the `example-crawl` collection.
+
+A new WACZ for the analysis run will be created in the resulting `example-qa` collection.
+
+By default, the QA crawl will visit all of the pages (as read from the source WACZ file(s)), however pages can further be limited by adding `--include` and `--exclude` regexes. The `--limit` flag will also limit how many pages are tested.
+
+The QA crawl will skip over any non-HTML pages.
```

Comment:
Would expand a bit on why this is?  Feel free to edit if anything I've written is inaccurate!

```suggestion
The analysis crawl will skip over any non-HTML pages such as PDFs which can be relied upon to be bit-for-bit identical as long as the resource was fully fetched.
```